packages/experimental/Switch/src/useSwitch.ts

```diff
@@ -48,9 +70,12 @@ export const useSwitch = (props: SwitchProps): SwitchInfo => {
     props: {
       ...pressable.props,
       accessible: true,
+      accessibilityLabel: accessibilityLabel ?? label,
       accessibilityRole: accessibilityRole ?? 'switch',
-      accessibilityState: { disabled: disabled, checked: checkedState }, // Needed for E2E Testing to detect toggle state, will provide a better implementation in a Accessibility PR
-      accessibilityActions: defaultAccessibilityActions, // Needed for E2E Testing to detect toggle state, will provide a better implementation in a Accessibility PR
+      accessibilityHint: accessibilityHint,
```

Comment:
Is there a reason for why you don't pass in rest props here?
For props like accessibilityHint and label that aren't overriden in here, it may be simpler to let them be passed into `...rest` set of props, and in here instead of reassigning them, pass in `...rest` as you did with pressable.props.

Reply:
I think I was trying to filter out props that were not part of the control. But I think with what you pointed out there are also other issues with doing that.  Thanks!